### ISSUE TITLE
Centering plot to my data in it

### ISSUE THREAD

**Issue by IvaOsi** [epezent/implot#286]
Hello, everyone! 
I have a little data, I draw it like this:

```
static ImPlotAxisFlags xflags = ImPlotAxisFlags_RangeFit;
static ImPlotAxisFlags yflags = ImPlotAxisFlags_RangeFit;
 if (ImPlot::BeginPlot("My Plot", "x_label","y_label", ImVec2(-1,0),0, xflags, yflags   )) {
            for (int i = 0; i < 10; ++i) {
                ImPlot::PlotText( "hello", 1, 100+i, false, ImVec2(0,0));
            }
            ImPlot::EndPlot();
        }
        ImGui::End();
```

The problem is that it takes me a very long time to scroll up the graph to look at my data on it.

![image](https://user-images.githubusercontent.com/17671542/132962248-0c56d468-d631-4bfd-b5ac-19bddbd3a0f8.png)
 
The temporary solution I have is to apply - `SetNextPlotLimits` But it imposes difficulties. Now I need to know in advance the limits of my data, which is an additional calculation. 

Also, it may not work with real-time data. How do I change the graph so that it automatically scales to my data and I see it instead of scrolling it?

**Comment by epezent** [comment]
I think you want `ImPlotAxisFlags_AutoFit` on your y-axis. Did you try that?

**Comment by IvaOsi** [comment]
> I think you want `ImPlotAxisFlags_AutoFit` on your y-axis. Did you try that?

its not working

**Comment by epezent** [comment]
Hi, the issue was that `PlotText` was not submitting text extents for fitting. A patch was included in #294.
